### PR TITLE
task9

### PR DESCRIPTION
**Pull request by tyzikova**

### CODE REVIEW

**Review by Killerkip10 — COMMENTED** [review]
## Task 1
- Please move trainings_pages folder to the root of your project;
- Good adaptive) (-5 points);
image
- (1.3) (-15 points);

### 25 points

## Task 2
- Bad code structure. (-5 points);
- What is it?) (-5 points);
- Why modal is not by center?)
image


### 45 points
## 70 points

File: task9/Trainings_Pages/Homework.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+</td>
+<td class="part_table">
+    <a class="colomn">Записаться</a>
+    <a href="#openModalJava" class="colomn2">Программа курса</a>
+<div id="openModalJava" class="modal">
+    <div>
+        <a href="#close" title="Close" class="close">&#10006;</a>
+    <h2>JavaScript</h2>
+    <hr>
+<div style="overflow: auto; height: 450px;">
```
Comment (Killerkip10) [inline task9/Trainings_Pages/Homework.html]: Why do you still use style attribute?

File: task9/Trainings_Pages/Homework.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        </td>
+<td>
+    <div><strong>859 BYN</strong></div>
+    <div>120 уч.ч (три месяца)</div>
+    <div><strong>Осталось</strong> 8 из 12 мест</div>
+</td>
+<td class="part_table">
+    <a class="colomn">Записаться</a>
+    <a href="#openModalJava" class="colomn2">Программа курса</a>
+<div id="openModalJava" class="modal">
```
Comment (Killerkip10) [inline task9/Trainings_Pages/Homework.html]: You can move all your modals to the bottom of document.

File: task9/Trainings_Pages/css/Styles.css [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+    content: "";
+    display: block;
+    height: 20px;
+    position: relative;
+    top: 73px;
+    background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0), #fff 75%)
+}
+.modal{
+    position: fixed;
+    font-family: Roboto;
```
Comment (Killerkip10) [inline task9/Trainings_Pages/css/Styles.css]: You have already set this font in the body

File: task9/Trainings_Pages/css/Styles.css [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+    position: absolute;
+    right: -5px;
+    text-align: left;
+    top: 5px;
+    width: 30px;
+    text-decoration: none;
+}
+@media only screen and (max-width: 800px){
+body{
+    font-size: 2.3vh;
```
Comment (Killerkip10) [inline task9/Trainings_Pages/css/Styles.css]: Why?)

File: task9/Trainings_Pages/Homework.html (hunk 1 above)
Comment (tyzikova) [reply task9/Trainings_Pages/Homework.html]: Corrected on class

File: task9/Trainings_Pages/css/Styles.css (hunk 3 above)
Comment (tyzikova) [reply task9/Trainings_Pages/css/Styles.css]: Deleted this font-family out .modal

File: task9/Trainings_Pages/css/Styles.css (hunk 4 above)
Comment (tyzikova) [reply task9/Trainings_Pages/css/Styles.css]: Corrected on px.

File: task9/Trainings_Pages/Homework.html (hunk 2 above)
Comment (tyzikova) [reply task9/Trainings_Pages/Homework.html]: not understand what you want from me((((

**Comment by Killerkip10** [comment]
You can merger this

**Pull request merged**